src/client.rs

```diff
@@ -736,6 +783,20 @@ fn current_time_millis() -> Result<u128, EsiError> {
     Ok(SystemTime::now().duration_since(UNIX_EPOCH)?.as_millis())
 }
 
+/// AuthenticationInformations contains informations needed to complete the requested authentication flow.
+pub struct AuthenticationInformations {
+    /// URL to call/pass to users to initiate an authentication and get an auth code from ESI.
+    pub authorization_url: String,
+    /// If the default feature "random_state" is enabled, the returned state field string will be
+    /// random; otherwise it'll be "rfesi_unused". The ESI docs link to [this auth0 page] to
```

Comment:
Missing URL for "this auth0 page"?

Reply:
Should be corrected. Probably a bad copy paste.